Sources/Structs/Unowned.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Unowned.swift
+//  HandySwift
+//
+//  Created by Murat Yilmaz on 19.05.18.
+//  Copyright © 2018 Flinesoft. All rights reserved.
+//
+
+import Foundation
+
+/// A wrapper for storing unowned references to a `Wrapped` instance.
+public struct Unowned<Wrapped> where Wrapped: AnyObject {
+    /// The value of `Wrapped` stored as unowned reference
+    public unowned var value: Wrapped
```

Comment:
Maybe we should name this unownedValue instead so the user knows immediately without looking at the type that it's unowned?